mex/common/primary_source/README.md

```diff
@@ -0,0 +1,21 @@
+Helper extractor to get metadata primary sources. It represents the original source of metadata that all the data in MEx will attach to. For example `confluence-vvt` primary source means: data extracted from `confluence-vvt` x-system will be attached to this primary source.
+
+## Common use cases
+
+- extract info of a particular primary source to attach extracted metadata to
+
+## Configuration
+
+To configure primary_source extractor, set `primary_sources_path` in settings to point to `primary-sources.json` file.
```

Comment:
we could mention the path to the sample file here: `assets/raw-data/primary-sources/primary-sources.json`